docs/docs/guide/automation/test.md

```diff
@@ -21,21 +21,17 @@ Test flakiness is a tremendous source of frustration for developers and loss of
 
 ## Running scheme tests
 
-To run the tests of a project, you can use the `tuist test` command. This command will generate the project if needed, and then run the tests using the `xcodebuild` command-line tool.
+To run the tests of a project, you can use the `tuist test` command. This command will generate the project if needed, and then run the tests using the `xcodebuild` command-line tool. We support the use of the `--` terminator to forward all subsequent arguments directly to `xcodebuild. Arguments such as `-workspace` or `-project` cannot be used because tuist takes care of them.
```

Comment:
I'd also update the [build documentation](https://docs.tuist.io/guide/automation/build.html)

Reply:
It's already [updated](https://github.com/tuist/tuist/pull/6300/files#diff-87e1c0a469c1dde85536f940ede004040c92f2af266dd0a1ef58def3a261ddc6)